watertap/examples/flowsheets/case_studies/wastewater_resource_recovery/dye_desalination/tests/test_dye_desalination_withRO.py

```diff
@@ -128,8 +128,8 @@ def test_costing(self, system_frame):
         assert_optimal_termination(results)
 
         # check costing
-        assert pytest.approx(0.653477, rel=1e-3) == value(m.fs.LCOW)
-        assert pytest.approx(0.2701535, rel=1e-3) == value(m.fs.LCOT)
+        assert pytest.approx(0.690498, rel=1e-3) == value(m.fs.LCOW)
+        assert pytest.approx(0.284743, rel=1e-3) == value(m.fs.LCOT)
```

Comment:
I'm really surprised the results changed this much... we should probably look into that

Reply:
The change is now in the 4th significant figure. I was running a different version of IDAES locally.